### PR TITLE
Feature/delete se coverages

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/backend/src/routes/basin_subzones.js [hunk 1]
```diff
@@ -123,42 +90,6 @@ module.exports = (errorHandler, basinSubzoneService) => {
     ),
   );
 
-  /**
```
Comment (erikasv) [inline services/backend/src/routes/basin_subzones.js:126]: Why didn't you delete the previous endpoint? It is not used from the frontend

Comment (erikasv) [reply services/backend/src/routes/basin_subzones.js:126]: this also applies for all geofences

File: services/backend/src/service/basin_subzone.js [hunk 2]
```diff
@@ -67,14 +67,14 @@ module.exports = (basinSubzonePersistence, seService) => {
      * @param {String} subzoneId subzone id
      * @param {String} seType strategic ecosystem type
      */
-    getCoverageInSE: async (subzoneId, seType) => {
+    /* getCoverageInSE: async (subzoneId, seType) => {
```
Comment (Trjegul84) [inline services/backend/src/service/basin_subzone.js:70]: In service there are also commented lines.

File: services/backend/src/service/basin_subzone.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ module.exports = (basinSubzonePersistence, seService) => {
     /**
      * Get subzone total area divided by strategic ecosystem type
      */
-    getAreaBySE: async (subzoneId) => {
+    /* getAreaBySE: async (subzoneId) => {
```
Comment (erikasv) [inline services/backend/src/service/basin_subzone.js:24]: Why did you commented instead of delete?

File: services/backend/api_doc/coverage.js [hunk 4]
```diff
@@ -20,20 +20,3 @@
  *    }
  *  ]
  */
-
```
Comment (erikasv) [inline services/backend/api_doc/coverage.js:23]: This file should be deleted, the first definition isn't being used

File: services/backend/api_doc/strategic_ecosystems.js [hunk 5]
```diff
@@ -11,30 +11,3 @@
  *    "national_percentage": 0.1523
  *  }
  */
-
```
Comment (erikasv) [inline services/backend/api_doc/strategic_ecosystems.js:14]: delete this file too, the endpoints that use `SEInGeofenceDetailExample` aren't being used by the front

**Review by erikasv — CHANGES_REQUESTED** [review]
check the comments

File: services/backend/src/service/strategic_ecosystem.js [hunk 6]
```diff
@@ -64,57 +39,6 @@ module.exports = (
     }
   },
 
-  /**
```
Comment (erikasv) [inline services/backend/src/service/strategic_ecosystem.js:67]: the previous function should be deleted too

File: services/backend/src/service/basin_subzone.js (hunk 2 above)
Comment (cazapatamar) [reply services/backend/src/service/basin_subzone.js]: Done!

File: services/backend/src/service/basin_subzone.js (hunk 3 above)
Comment (cazapatamar) [reply services/backend/src/service/basin_subzone.js]: Deleted!

File: services/backend/src/persistence/paramo.js [hunk 7]
```diff
@@ -1,40 +1,4 @@
...
-   *
-   * @param {String} stateId state id
-   * @param {Number} year optional year to filter data, 2012 by default
-   */
-  findAreaByState: (stateId, year = 2012) =>
-    geoParamoDetails
-      .query()
-      .where({ id_state: stateId, year_cover: year })
-      .select(db.raw('coalesce(SUM(area_ha), 0) as area')),
-
```
Comment (erikasv) [inline services/backend/src/persistence/paramo.js:37]: These functions are used by `getSEAreaInEA` in strategic ecosystem service.

Comment (cazapatamar) [reply services/backend/src/persistence/paramo.js]: Fixed!

File: services/backend/src/persistence/tropicalDryForest.js [hunk 8]
```diff
@@ -1,40 +1,4 @@
...
-   *
-   * @param {String} stateId state id
-   * @param {Number} year optional year to filter data, 2012 by default
-   */
-  findAreaByState: (stateId, year = 2012) =>
-    geoTropicalDryForestDetails
-      .query()
-      .where({ id_state: stateId, year_cover: year })
-      .select(db.raw('coalesce(SUM(area_ha), 0) as area')),
-
```
Comment (erikasv) [inline services/backend/src/persistence/tropicalDryForest.js:37]: The same as these functions in paramo, and also for wetland

Comment (cazapatamar) [reply services/backend/src/persistence/tropicalDryForest.js]: Done!

**Review by erikasv — APPROVED** [review]

**Review by Trjegul84 — APPROVED** [review]

**Pull request merged**